model/bitmap-constructor.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.BufferData = function(data) {
+  this.buffer = data;
+  this.id = data.toString('utf-8', 0, 2);
+  this.size = data.readInt32LE(2);
+  this.offset = data.readInt32LE(10);
+  this.dibheader = data.readInt32LE(14);
+  this.width = data.readInt32LE(18);
+  this.height = data.readInt32LE(22);
+  this.pixel_horizontal = data.readInt32LE(38);
+  this.pixel_vertical = data.readInt32LE(42);
+  this.number_colors = data.readInt32LE(46);
+  this.colorArray = data.slice(54, this.offset);
```

Comment:
@brittdawn @jonathanheemstra @peterkim2 **excellent** bitmap constructor - I like that you included the raw buffer in your constructor, along with storing all of the necessary metadata about the bitmap